.zenodo.json

```diff
@@ -0,0 +1,96 @@
+{
+  "title": "MNE-BIDS: Organizing MEG, EEG, and iEEG data according to the BIDS specification and facilitating their analysis with MNE-Python.",
+  "description": "MNE-BIDS is a Python library to facilitate the analysis of MEG, EEG, and iEEG data with MNE-Python for data conforming to the BIDS (Brain Imaging Data Structure) format. More information about BIDS and MNE can be found under <a href=\"https://bids.neuroimaging.io\">bids.neuroimaging.io</a> and <a href=\"https://mne.tools\">mne.tools</a> respectively.",
+  "creators": [
+    {
+      "affiliation": "Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany",
+      "name": "Appelhoff, Stefan",
```

Comment:
Most people here used the first name last name format, this should be made consistent.